external-workshops/using-tools-and-materials.md

```diff
@@ -0,0 +1,38 @@
+# Using tools and materials
+
+**You must have GitHub, GitKraken, R, RStudio, and a suitable text editor installed on the machine you will be instructing with.**
+You should do this before proceeding with the instructions below.
+The [`setup_instructions` folder](https://github.com/AlexsLemonade/reproducible-research/tree/main/docs/setup_instructions) in the `reproducible-research` repository contains helpful information for installing these tools. 
+
+## Training materials
+
+**1.) The [`reproducible-research` repository](https://github.com/AlexsLemonade/reproducible-research) holds materials such as lecture slides, cheatsheets, and instructor notes.**
+Fork the repository, to create your own copy of the materials.
+
+* Navigate to the [`reproducible-research` repository](https://github.com/AlexsLemonade/reproducible-research) and click `Fork`.
+<img src="screenshots/reproducible-research-header-fork-1.png" width=800>
+
```

Comment:
Spacing is a little tight here when I actually look at the file, so let's force a new line - 

```suggestion
<br>
```